src/controllers/assets/AssetsController.ts

```diff
@@ -0,0 +1,38 @@
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+
+import { AssetsService } from '../../services';
+import AbstractController from '../AbstractController';
+
+export default class AssetsController extends AbstractController<AssetsService> {
+	constructor(api: ApiPromise) {
+		super(api, '/assets', new AssetsService(api));
+		this.initRoutes();
+	}
+
+	protected initRoutes(): void {
+		this.safeMountAsyncGetHandlers([
+			['/:assetsId/assetInfo', this.getAssetById],
```

Comment:
```suggestion
			['/:assetsId/asset-info', this.getAssetById],
```
kebab-case is consistent with all out other URIs